commons/src/components/NError.svelte

```diff
@@ -29,7 +30,7 @@
   }
 </style>
 
-{#if error.name}
+{#if error.name && isDevEnv}
```

Comment:
Hey @heathergh can you test if there is an error like `invalid component id`? I believe in such cases, we don't get a `name` property back so this error never gets displayed.

Reply:
done!